### PR TITLE
Bip120

### PR DESCRIPTION
**Pull request by crushurh3aet**

### CODE REVIEW

**Comment by kallerosenbaum** [comment]
Closed my original pull-req in favor of this one.

**Comment by schildbach** [comment]
Just a comment: there is no reason to close PR and open new ones for the same branch. Just force-push over your existing branch and GitHub will pick up the changes.

File: wallet/AndroidManifest.xml [hunk 1]
```diff
@@ -180,7 +180,29 @@
...
+				<category android:name="android.intent.category.BROWSABLE" />
+			</intent-filter>
+			<intent-filter android:label="@string/pop_activity_title" >
+				<action android:name="android.nfc.action.NDEF_DISCOVERED" />
 
+				<data android:scheme="btcpop" />
+
+				<category android:name="android.intent.category.DEFAULT" />
+			</intent-filter>
+		</activity>
```
Comment (dffp) [inline wallet/AndroidManifest.xml]: ?

**Comment by schildbach** [comment]
I'm afraid BIP120 hasn't gained any momentum in the industry. This unfortunately means the feature would not be of much use for Bitcoin Wallet users. So I'm closing this for now.

**Comment by kallerosenbaum** [comment]
Thanks, I had forgotten about this PR. FYI I have withdrawn the BIPs 120 and 121 because of potential problems during soft forks. Thanks.

**Pull request closed without merging**